### ISSUE TITLE
Parsing of packages fails for most package strings (requirement)

### ISSUE THREAD

**Issue by matthiaskoenig** [Midnighter/dependency-info#5]
#### Problem description
Most package strings which can occur in setup.cfg are currently not parsed correctly by the package resulting in "not installed".
E.g.
```
        ("pip", "pip"),
        ("pip (~=1.4)", "pip"),
        ("pip (==5.18.3)", "pip"),
        ("pip (<1.4.6)", "pip"),
        ("pip (>1.4.6)", "pip"),
        ("pip>=2.0.5", "pip"),
        ("pip==1.19.3", "pip"),
        ('pip>=6.1; extra == "development"', "pip"),
        ('pip>=6.1 ; extra == "development"', "pip"),
```
Pull request with fix comes in a minute.

**Comment by Midnighter** [comment]
Just released 1.7.0 @matthiaskoenig

**Comment by matthiaskoenig** [comment]
@Midnighter Thanks for the quick release ;)
